### PR TITLE
Rubocop whitespace fix

### PR DESCRIPTION
**Pull request by ollie**
Hello, I trimmed the trailing whitespace and fixed the indentation. It looks like a lot of changes but if you `git diff 529cfc913b7939ee4330df099d189b65aceaab50..HEAD -w` you will see nothing was changed otherwise apart from `.rubocop.yml`. :smile:

### CODE REVIEW

File: test/Image2.rb [hunk 1]
```diff
@@ -10,46 +10,46 @@ class Image2_UT < Test::Unit::TestCase
...
     def test_composite!
-        img1 = Magick::Image.read(IMAGES_DIR+'/Button_0.gif').first
-        img2 = Magick::Image.read(IMAGES_DIR+'/Button_1.gif').first
-        assert_nothing_raised do
-            res = img1.composite!(img2, Magick::NorthWestGravity, Magick::OverCompositeOp)
-            assert_same(img1, res)
-        end
-        img1.freeze
-        assert_raise(FreezeError) { img1.composite!(img2, Magick::NorthWestGravity, Magick::OverCompositeOp) }
+      img1 = Magick::Image.read(IMAGES_DIR+'/Button_0.gif').first
```
Comment (vassilevsky) [inline test/Image2.rb]: Could you tweak the config to suppress these messages? We prefer single quotes like in the community guide.

**Comment by vassilevsky** [comment]
https://github.com/gemhome/rmagick/pull/196/files?w=1 true

File: test/Image2.rb (hunk 1 above)
Comment (ollie) [inline test/Image2.rb]: Ah, sure, I deleted it along with the old file. It is now added back on top.

**Comment by vassilevsky** [comment]
Thank you! Whitespace is now eliminated ✂️

**Pull request merged**